src/hooks/useReloadOnResize.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from "react";
+
+const useReloadOnResize = (): void => {
+
+    useEffect(() => {
+        const documentWidth = document.documentElement.clientWidth;
+        let time: any;
+        window.onresize = function () {
+            if (time)
+                clearTimeout(time);
+            time = setTimeout(function () {
+                if (documentWidth !== document.documentElement.clientWidth) {
+                    window.location.reload();
```

Comment:
I dont see reload() as an option here to be honest, we need to elaborate another solution